package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ibm-continuous-delivery",
+  "name": "@ibm-cloud/continuous-delivery",
```

Comment:
The default "access" for a non-scoped project is public, while the default for a scoped package is private. Therefore, you should add this to your package.json:
```
  "publishConfig": {
    "access": "public"
  },
```
Also, just a reminder to make sure that the NPM id being used by your travis build to publish releases has been given the proper authority in the `@ibm-cloud` npm org (you request this by opening an issue per the instructions re: the use of the `@ibm-cloud` scope).

Reply:
Thanks, added that to package.json

@omar-albastami tells me that the npm token is for the ids-org user, so we should be good to go on that front

Reply:
@dpopp07 Does the "ids-org" npm user have the authority to publish an `@ibm-cloud`-scoped package?  Just wanted to double check.

Reply:
The username is `idsorg` but yes, they are a part of the org 👍